apps/desktop-wallet/src/features/assetsLists/types.ts

```diff
@@ -20,18 +20,15 @@ import { AddressHash } from '@alephium/shared'
 
 export type TokensTabValue = 'fts' | 'nfts' | 'nsts'
 
-export interface AssetsTabsProps {
+export type TokensTabsBaseProps = {
```

Comment:
> Let's use interfaces only?

We can't use `interfaces` only, sometimes we have to use types. But in this example we can indeed use `interface`. A reason to use `type` instead of `interface` is when you want to merge 2 interfaces/types together.:

```ts
type A = {}
type B = {}

// with type:
type ABType = A & B

// with interface:
interface ABInterface extends A, B {}
```

In the above case, I prefer `type` over `interface` cause we don't need that ugly `{}` at the end.

Reply:
Fixed in this PR!
- https://github.com/alephium/alephium-frontend/pull/842

Reply:
Thanks! Yeah I know the differences between the 2, I was really talking about this specific case :) Because we usually use interfaces for Props.